setup.py

```diff
@@ -129,9 +132,14 @@ def py2exe_options():
     author_email=author_email,
     classifiers=filter(None, classifiers.split('\n')),
     license=license,
-    scripts=['openerp-server', 'odoo.py'],
+    scripts=['openerp-server'],
+    entry_points={
+        'console_scripts': [
+            'odoo-bin=odoo.cli.odoobin:main',
```

Comment:
@sbidoul @rco-odoo Shouldn't this be `odoo=odoo.cli.odoobin:main`? Our idea would be that when you do a local installation of Odoo you get to use the simplest `odoo` command everywhere, à la `rails`. This way you can conveniently run `odoo shell -d test` from anywhere.
On the other hand if you only do a `git clone` you'll still be able to directly run odoo with the same parameters/API, using `./odoo-bin` instead of `odoo` (that's the closest we can do, I guess). E.g `./odoo-bin shell -d test`